src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -1,27 +1,38 @@
 import React, { useEffect, useState } from 'react'
 import { useParams } from 'react-router-dom'
 
-import { useChannel, useFollowChannel, useUnfollowChannel } from '@/api/hooks'
-import { InfiniteVideoGrid, ViewWrapper } from '@/components'
+import { useChannel, useFollowChannel, useUnfollowChannel, useVideosConnection } from '@/api/hooks'
+import { VideoOrderByInput } from '@/api/queries'
+import { LimitedWidthContainer, VideoPreview, ViewWrapper } from '@/components'
 import { AssetType, useAsset, usePersonalDataStore } from '@/providers'
-import { Button, ChannelCover } from '@/shared/components'
+import { ChannelCover, Grid, Pagination, Select, Tabs, Text } from '@/shared/components'
+import { SvgGlyphCheck, SvgGlyphPlus } from '@/shared/icons'
 import { transitions } from '@/shared/theme'
 import { Logger } from '@/utils/logger'
 import { formatNumberShort } from '@/utils/number'
+import { SORT_OPTIONS } from '@/views/studio/MyVideosView/MyVideosView'
```

Comment:
We should never import into view from another view as this can easily create unexpected import hierarchy. If `SORT_OPTIONS` is needed app-wide, let's extract it from My Videos and define in some generic place